models.py

```diff
@@ -53,6 +53,33 @@ def serialize(self):
         }
 
 
+class Chapter(db.Model):
+    __tablename__ = 'ChapterData'
+
+    def serialize(self):
+        return {
+            'chapterNumber': str(self.babID),
+            'chapter': [
+                {
+                    'lang': 'ar',
+                    'bookId': str(self.arabicBookID),
+                    'chapterNumber': str(self.arabicBabNumber),
+                    'chapterTitle': self.arabicBabName,
+                    'intro': self.arabicIntro,
+                    'ending': self.arabicEnding
+                },
+                {
+                    'lang': 'en',
+                    'bookId': str(self.englishBookID),
```

Comment:
This field will be redundant when the other field is moved outside the language-specific block.